Src/WitsmlExplorer.Frontend/components/ContentViews/JobsView.tsx

```diff
@@ -140,17 +177,16 @@ export const JobsView = (): React.ReactElement => {
             wellName: jobInfo.wellName,
             wellboreName: jobInfo.wellboreName,
             objectName: jobInfo.objectName,
-            status:
-              jobInfo.progress && jobInfo.status === JobStatus.Started
-                ? `${Math.round(jobInfo.progress * 100)}%`
-                : jobInfo.status,
+            status: getJobStatus(jobInfo, cancellingJobs),
             cancel:
-              jobInfo.isCancelable === true && jobInfo.status === "Started" ? (
+              jobInfo.isCancelable === true &&
+              jobInfo.status === JobStatus.Started &&
+              getJobStatus(jobInfo, cancellingJobs) !== "Cancelling" ? (
```

Comment:
Maybe a bit cleaner to replace the call to getJobStatus with `!cancellingJobs.includes(jobInfo.id)`?

Reply:
done